gef-ui/server/controllers/about-facility/validation.js

```diff
@@ -304,7 +241,7 @@ const validateAboutFacility = ({
   }
 
   if (isFacilityEndDateEnabled) {
-    if (!isUsingFacilityEndDate && !saveAndReturn) {
+    if (isUsingFacilityEndDate === undefined && !saveAndReturn) {
```

Comment:
What's the reason for changing this from `!isUsingFacilityEndDate`?

Reply:
`isUsingFacilityEndDate` can be `false` & we should only bypass the validation if no value has been provided